test failure could be related
```
======================================================================
ERROR: corehq.apps.reminders.tests.test_recipient:ReminderRecipientTest.test_recipient_case_owner_location_parent
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 267, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 238, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 253, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 234, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/reminders/tests/test_recipient.py", line 75, in test_recipient_case_owner_location_parent
    self.user.unset_location()
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1870, in unset_location
    self.save()
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1528, in save
    super(CommCareUser, self).save(**params)
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1394, in save
    super(CouchUser, self).save(**params)
  File "/vendor/lib/python2.7/site-packages/couchdbkit/schema/base.py", line 134, in save
    db.save_doc(doc, **params)
  File "/vendor/lib/python2.7/site-packages/couchdbkit/client.py", line 516, in save_doc
    **params).json_body
  File "/vendor/lib/python2.7/site-packages/restkit/resource.py", line 153, in put
    headers=headers, params_dict=params_dict, **params)
  File "/vendor/lib/python2.7/site-packages/couchdbkit/resource.py", line 141, in request
    response=e.response)
ResourceConflict: Document update conflict.
-------------------- >> begin captured stdout << ---------------------
<function test_recipient_case_owner_location_parent at 0x7f2feed13398> failed with the following settings:
settings.TESTS_SHOULD_USE_SQL_BACKEND = False
--------------------- >> end captured stdout << ----------------------
```

--- a/corehq/apps/reminders/tests/test_recipient.py
+++ b/corehq/apps/reminders/tests/test_recipient.py
@@ -65,14 +65,13 @@ class ReminderRecipientTest(TestCase):
                 self.assertEqual(reminder.recipient, [parent_location])
 
             # Remove parent location
-            parent_location.delete()
             child_location.parent = None
             child_location.save()
+            parent_location.delete()
             with patch('corehq.apps.reminders.models.CaseReminder.handler', new=handler):
                 self.assertIsNone(reminder.recipient)
 
             # Remove child location
-            self.user = CommCareUser.get(self.user._id)
             self.user.unset_location()
             child_location.delete()
             with patch('corehq.apps.reminders.models.CaseReminder.handler', new=handler):

--- a/corehq/apps/reminders/tests/test_recipient.py
+++ b/corehq/apps/reminders/tests/test_recipient.py
@@ -72,6 +72,7 @@ class ReminderRecipientTest(TestCase):
                 self.assertIsNone(reminder.recipient)
 
             # Remove child location
+            self.user = CommCareUser.get(self.user._id)
             self.user.unset_location()
             child_location.delete()
             with patch('corehq.apps.reminders.models.CaseReminder.handler', new=handler):

--- a/corehq/apps/reminders/tests/test_recipient.py
+++ b/corehq/apps/reminders/tests/test_recipient.py
@@ -44,6 +44,7 @@ class ReminderRecipientTest(TestCase):
             site_code='parent',
             location_type=self.parent_location_type
         )
+        self.addCleanup(parent_location.delete)
 
         child_location = SQLLocation.objects.create(
             domain=self.domain,
@@ -52,6 +53,7 @@ class ReminderRecipientTest(TestCase):
             location_type=self.child_location_type,
             parent=parent_location
         )
+        self.addCleanup(child_location.delete)
 
         self.user.set_location(child_location)
 
@@ -67,13 +69,11 @@ class ReminderRecipientTest(TestCase):
             # Remove parent location
             child_location.parent = None
             child_location.save()
-            parent_location.delete()
             with patch('corehq.apps.reminders.models.CaseReminder.handler', new=handler):
                 self.assertIsNone(reminder.recipient)
 
             # Remove child location
             self.user.unset_location()
-            child_location.delete()
             with patch('corehq.apps.reminders.models.CaseReminder.handler', new=handler):
                 self.assertIsNone(reminder.recipient)
 

